src-docs/src/views/selection_controls/guidelines.js

```diff
@@ -209,12 +303,117 @@ export default () => {
+            }}
+          />
+        </GuideRuleExample>
+        <GuideRuleExample
+          type="dont"
+          text="Mix label structure within the same group of options."
+        >
+          <EuiCheckboxGroup
+            options={groupTitleDont}
+            onChange={() => {}}
+            idToSelectedMap={{ group_title_dont_one: true }}
+            legend={{
+              children: <span>Data to ship</span>,
+            }}
+          />
+        </GuideRuleExample>
+      </GuideRule>
+
+      <GuideRule>
+        <GuideRuleExample
+          type="do"
+          text="Start all with a verb, all with a noun, or all with a number."
+        >
+          <EuiRadioGroup
+            options={consistentDo}
+            idSelected={{ consistent_do_one: true }}
+            onChange={() => {}}
+            name="radio group"
+            legend={{
+              children: <span>Availability zones</span>,
+            }}
+          />
+        </GuideRuleExample>
+        <GuideRuleExample
+          type="dont"
+          text="Use ending punctuation. 
+          Exception: Question marks can be used if they add clarity."
+        >
+          <EuiRadioGroup
+            options={consistentDont}
+            idSelected={{ consistent_dont_one: true }}
+            onChange={() => {}}
+            name="radio group"
+            legend={{
+              children: <span>Availability zones</span>,
+            }}
+          />
+        </GuideRuleExample>
+      </GuideRule>
+
+      <GuideRule
+        heading="Legends"
+        description={
+          <p>
+            For groups of options, add a legend that gives meaning to the
+            options. Legends appear like a title.
+          </p>
+        }
+      >
+        <GuideRuleExample
+          type="do"
+          text="Use nouns that describe the choices to make."
+        >
+          <EuiCheckboxGroup
+            options={groupTitleDo}
+            onChange={() => {}}
+            idToSelectedMap={{ group_title_do_one: true }}
+            legend={{
+              children: <span>Data to ship</span>,
+            }}
+          />
+        </GuideRuleExample>
+        <GuideRuleExample
+          type="dont"
+          text="Repeat choices or use the title as instructions."
+        >
+          <EuiCheckboxGroup
+            options={groupTitleDo}
+            onChange={() => {}}
+            idToSelectedMap={{ group_title_do_one: true }}
+            legend={{
+              children: <span>Choose to ship logs and/or metrics</span>,
+            }}
+          />
+        </GuideRuleExample>
+      </GuideRule>
+
       <GuideRule
         heading="Switch labels"
         description={
           <p>
-            The label should be static, action-oriented and describe the
-            feature.
+            The label should be static, action-oriented, and describe the
+            feature. Avoid verbs that are less conversational such as `enable`
```

Comment:
We normally use the following quotation mark in our guidelines:
```suggestion
            feature. Avoid verbs that are less conversational such as "enable"
```